src/main/java/io/strimzi/kafka/bridge/http/HttpSourceBridgeEndpoint.java

```diff
@@ -169,7 +133,8 @@ public Iterator<Map.Entry<String, String>> iterator() {
         List<Future> sendHandlers = new ArrayList<>(records.size());
         for (KafkaProducerRecord<K, V> record : records) {
             Promise<RecordMetadata> promise = Promise.promise();
-            sendHandlers.add(promise.future());
+            Future<RecordMetadata> future = promise.future();
```

Comment:
why this change about adding a `future` variable?

Reply:
I needed it before for that callback hack -- removed now.